.codecov.yml

```diff
@@ -1,6 +1,6 @@
 coverage:
   precision: 2
-  range: "60…100"
+  range: "60...100"
```

Comment:
This defines just the range of color used. Try the config below:
```
coverage:
  precision: 2
  round: down
  range: "55...95"

  status:
    project:
      default:
        target: "60%"
    patch:
      default: off
    changes:
      default: off

comment:
  layout: "header, reach, diff, flags, files, footer"
  behavior: default
  require_changes: no
  require_base: no
  require_head: yes
```

Reply:
Can do!